.travis.yml

```diff
@@ -42,11 +14,8 @@ rvm:
   - 2.1
   - 2.2
   - ree
+  - rbx-2
 matrix:
-  allow_failures:
-    - env: DB=mysql57
-    - rvm: rbx-2
-    - os: osx
```

Comment:
so you want to continue to allow those failures? I haven't seen any spurious failures during my work here...